ocs_ci/ocs/resources/mcg.py

```diff
@@ -97,6 +98,19 @@ def __init__(self):
                 aws_access_key_id=self.aws_access_key_id,
                 aws_secret_access_key=self.aws_access_key
             )
+            logger.info('Checking whether RGW pod is not present on AWS platform')
+            pods = pod.get_rgw_pods()
+            assert len(pods) == 0, 'RGW pod is should not exist on AWS platform'
```

Comment:
RGW pod should not exist on AWS platform

Reply:
ack